src/neuroglancer/sliceview/png/index.ts

```diff
@@ -160,9 +161,9 @@ export async function decompressPng(
   }
 
   if (
-    sx !== width 
-    || sy !== height 
-    || numComponents !== numChannels
+    width && sx !== width 
```

Comment:
As elsewhere, explicitly check for `!== undefined`

Reply:
Done in https://github.com/google/neuroglancer/pull/453/commits/a5737feb51489a7b612d8bee5a376612ef25b608